tempesta_fw/addr.c

```diff
@@ -382,9 +382,6 @@ tfw_addr_ifmatch(const TfwAddr *server, const TfwAddr *listener)
 		return 0;
 	}
 
-	if (tfw_addr_port(listener) != tfw_addr_port(server))
```

Comment:
This comparison is needed for the case where the code below returns `1`. If listening address is `0.0.0.0` and server address is in loopback subnet, we need to be sure that ports match. And only then it's a closed loop. Otherwise it's fine: we may serve at 0.0.0.0:80 while having Nginx at 127.0.0.1:81 as a backend.

Reply:
You're right. Thanks!